projects/ghc-performance-dashboards.markdown

```diff
@@ -0,0 +1,14 @@
+---
+status: ideation
+title: GHC Performance Dashboards
+summary: The Haskell Foundation is working with the GHC team to help provide dashboard monitors for Haskell critical libraries (e.g. Aeson, Text, Bytestring).
```

Comment:
I think if we're not going to have stand alone pages then the 'summary' can be the body text, which would also allow us to style it if appropriate.

Reply:
I was thinking from the opposite :smile: 

If we want dedicated pages about each projects : the current previews are generated from metadata and each detailed page will be from body.

I'm taking a look, I spotted a problem with contexts and body in this part

Reply:
Done (if I had a good understanding of your wish :wink:)